efficientdet/keras/train_lib.py

```diff
@@ -488,12 +488,12 @@ def __init__(self, delta=0.1, **kwargs):
   def call(self, y_true, box_outputs):
     num_positives, box_targets = y_true
     normalizer = num_positives * 4.0
-    mask = tf.cast(box_targets != 0.0, num_positives.dtype)
+    mask = tf.cast(box_targets != 0.0, box_targets.dtype)
```

Comment:
num_positives could be a scalar (has no .dtype), so I changed it to box_targets.

Reply:
I changed it in another PR too.